plugins/builds/triggers/orBase.js

```diff
@@ -56,12 +56,16 @@ class OrBase {
                 return nextBuild;
             }
 
+            nextBuild.parentBuildId = [this.currentBuild.id];
```

Comment:
```suggestion
            nextBuild.parentBuildId = this.currentBuild.id;
```
Should this be a singular item?

Reply:
No. Definition of `parentBuildId` is array.

https://github.com/screwdriver-cd/data-schema/blob/714afa62b464326b28a9fe3bc82cffba14e54b41/models/build.js#L36